x-pack/plugins/session_view/public/hooks/use_process_tree.ts

```diff
@@ -146,16 +146,21 @@ class Process implements IProcess {
   }
 
   getMaxAlertLevel() {
-    //TODO:
+    // TODO:
     return null;
   }
 }
 
-type ProcessMap = {
+interface ProcessMap {
```

Comment:
so, it's also, an Eslint rule, which I very much disagree with, because in some cases types are better than an interface, otherwise they wouldn't exist in the first place, 

I found that in many other plugins only this rule is being overridden, by creating a`.eslintrc.json` in the root folder of the plugin, with the following:

```json
{
  "rules": {
    "@typescript-eslint/consistent-type-definitions": 0
  }
}
```

Think we can do the same

Reply:
I pushed a custom `eslintrc.json` file with that rule override and reverted back that interface to type. Think with time we can review other things that are very opinionated

Reply:
i mean, if it works it works, but yea..